docs/testcases/UI/HCS/2749.md

```diff
@@ -0,0 +1,28 @@
+# [Manual] Ability to merge Z-stack in a single projection
+
+Test verifies
+- that User has ability to merge several Z-planes of the same HCS image (in each field) in a single image (intensity projection)
+
+**Prerequisites**:
+- User that has full permissions to the storage with HCS images
+- HCS image with several Z-planes
+
+| Steps | Actions | Expected results |
+| :---: | --- | --- |
+| 1 | Login as the user from the prerequisites | |
+| 2 | Open the **Library** page | |
+| 3 | Find HCS image from the Prerequisites | |
+| 4 | Click the **HCS** icon near the HCS image file name | ***Preview pop-up*** opens that contains <li> All available image Z-planes are shown in the *Z-planes* section: <ul><li> first Z-plane is selected by default <li> *Select all* link is shown |
+| 5 | Click **Analysis** icon on the *Preview panel* | |
+| 6 | Select any field on the *Well panel* | |
+| 7 | Click *Select all* link | Z-planes panel: <li> All z-planes are selected <li> *Use planes projection* checkbox appears <li> *Select all* link disappears </li> Analysis panel: <li> Message `Selection: 1 well, 1 field, 1 time point, <n> z-planes. Analysis will be submitted as a batch job that can be found at Evaluations tab. For fine-tuning pipeline select a single well, field, and time point.` is shown on the Analysis panel |
+| 8 | Tick *Use planes projection* checkbox | Message about batch job disappears from Analysis panel |
+| 9 | Click 2nd z-plane | <li> 2nd z-plane is unselected <li> *Select all* link appears |
```

Comment:
Fix to `Holding Shift key, click 2nd z-plane...`

Reply:
Done